[INFO   ] Logger: Record log in /storage/emulated/0/Controle financeiro v0.1/.kivy/logs/kivy_22-12-01_17.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  9 2021, 08:02:40) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] KivyMD: 1.1.1, git-Unknown, 2022-11-05 (installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivymd/__init__.py")
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[CRITICAL] Window: Unable to find any valuable Window provider. Please enable debug logging (e.g. add -d if running from the command line, or change the log level in the config) and re-run your app to identify potential causes
sdl2 - ValueError: invalid literal for int() with base 10: '"degrees"'
  File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/core/__init__.py", line 70, in core_select_lib
    cls = cls()
  File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/core/window/window_sdl2.py", line 152, in __init__
    super(WindowSDL, self).__init__()
  File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivy/core/window/__init__.py", line 936, in __init__
    kwargs['rotation'] = Config.getint('graphics', 'rotation')
  File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/configparser.py", line 818, in getint
    return self._get_conv(section, option, int, raw=raw, vars=vars,
  File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/configparser.py", line 808, in _get_conv
    return self._get(section, conv, option, raw=raw, vars=vars,
  File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/configparser.py", line 803, in _get
    return conv(self.get(section, option, **kwargs))

[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Controle financeiro v0.1/main.py", line 12, in <module>
[WARNING] stderr:     from kivymd.uix.datatables import MDDataTable
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivymd/uix/__init__.py", line 8, in <module>
[WARNING] stderr:     from kivymd.uix.behaviors import SpecificBackgroundColorBehavior
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivymd/uix/behaviors/__init__.py", line 8, in <module>
[WARNING] stderr:     from .backgroundcolor_behavior import (
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivymd/uix/behaviors/backgroundcolor_behavior.py", line 26, in <module>
[WARNING] stderr:     from kivymd.theming import ThemeManager
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivymd/theming.py", line 235, in <module>
[WARNING] stderr:     from kivymd.material_resources import DEVICE_IOS, DEVICE_TYPE
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.9/site-packages/kivymd/material_resources.py", line 20, in <module>
[WARNING] stderr:     elif Window.width >= dp(600) and Window.height >= dp(600):
[WARNING] stderr: AttributeError: 'NoneType' object has no attribute 'width'
